mainwindow.cpp

```diff
@@ -418,9 +409,16 @@ void MainWindow::on_actionCopyToClipboard_triggered()
     if (!m_hashesTable)
         return;
     QModelIndexList indexes = m_ui->tableView_Hashes->selectionModel()->selectedIndexes();
-    // TODO: maybe copy everything if nothing is selected?
```

Comment:
Let's get rid of the exclamation mark. Only use it when something really needs emphasizing, otherwise just terminate with a period.